recipes/RNAtools/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "RNAtools" %}
+{% set version = "0.8.4" %}
+{% set sha256 = "6e6285c54bba3e492581c3b90b2dd86e1f2c2605c6298bee5280fdf0ce19a39b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - RNAtools
+    - RNAtools.graph
+
+about:
+  home: http://github.com/grice/RNAtools
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'RNAtools is a Python package for the manipulation of RNA files.'
+
+extra:
+  recipe-maintainers:
```

Comment:
Can you check with the listed co-maintainers to make sure they're ok with being added here, or just let us know if you already had that conversation.

Reply:
@synapticarbors we've already had that conversation, so no problems here. Thanks for the reminder nonetheless!